docs/project/dogfooding.md

```diff
@@ -6,6 +6,23 @@ This document provides the steps necessary to consume a nightly build of
 Please note that these steps are likely to change as we're simplifying
 this experience. Make sure to consult this document often.
 
+## Obtaining nightly builds of NuGet packages
+
+If you are only looking to get fixes for an individual NuGet package, and don't need a preview version of the entire runtime, you can add the following package feed to `NuGet.config`:
```

Comment:
Adding this feed won't automatically give you any fixes. You also need to update the versions of the NuGet packages.